CONTRIBUTING.md

```diff
@@ -145,8 +145,10 @@ Cluster API maintains the most recent release/releases for all supported API and
 | **v1alpha4** | EOL since 2022-04-06 |
 | **v1alpha3** | EOL since 2022-02-23 |
 
-- For the latest API version we support the two most recent minor releases; older minor releases are immediately unsupported when a new major/minor release is available. 
+- For the latest API version we support the two most recent minor releases; older minor releases are immediately unsupported when a new major/minor release is available.
```

Comment:
```suggestion
- For the current stable API version (v1beta1) we support the two most recent minor releases; older minor releases are immediately unsupported when a new major/minor release is available.
```
s/latest/current stable because it is the same wording used in the table above
(we can also change the table and keep latest here and below, up to you)